sdk/rust/oak_config_serializer/src/main.rs

```diff
@@ -57,18 +57,18 @@ struct Config {
 
 #[derive(Debug, serde::Deserialize)]
 #[serde(deny_unknown_fields)]
-enum Module {
-    #[serde(rename = "path")]
-    Path(String),
-    #[serde(rename = "external")]
-    External(External),
+struct Module {
+    file: WasmFile,
+    sha256: Option<String>,
```

Comment:
I'm not sure about this change; IIUC, this allows omitting the hash for remote files, which I think we should actively discourage, and ideally forbid entirely. For local files instead, there is not much value in having an explicit hash anyways, since it usually means that the module was already built locally, or comes from a trusted source.

Personally I would leave the interface as it was, and then perhaps make more structural change (e.g. adding the hash to local files too) in a separate, dedicated PR.

Reply:
Done